README.md

```diff
@@ -37,6 +37,7 @@
 
  - (September 9 - 10) [RustConf](http://rustconf.com/) (Portland, OR) - A day of trainings and a day of talks on the Rust programming language
  - (September 12 - 18) [Techweek: Kansas City](http://techweek.com/kansascity/) - The nation's leading technology conference and festival - showcasing entrepreneurship and innovation.
+ - :star: (September 15 - 16) [SymfonyLive](http://london2016.live.symfony.com/) (London, UK) - best practices, experience, knowledge and the latest developments with the framework
```

Comment:
@mAiNiNfEcTiOn that's how consumers know that it is a newly added list.  See the contribution guidelines on the readme, including the :star: is the right format. Thanks!